MyPet_en.properties

```diff
@@ -15,51 +15,51 @@ Message.CaptureHelper.NotUserCreated=This Iron Golem <red>wasn't<reset> build by
 Message.Command.CaptureHelper.Mode=CaptureHelper: <gold>{0}<reset>.
-Message.Command.ChooseSkilltree.OnlyOnce=You can change the skilltree of <aqua>{0}<reset> only <red>once<reset>!
+Message.Command.ChooseSkilltree.OnlyOnce=You can only change the skilltree of <aqua>{0}<reset> <red>once<reset>!
 Message.Command.ChoseSkilltree.AutomaticSkilltreeAssignment=Skilltrees will be assigned automatically!
 Message.Command.Fail=<red>Command execution failed!
 Message.Command.Help.Admin=See Wiki (link below)
 Message.Command.Help.Beacon=Opens the beacon window of the pet  (alias: /pbeacon)
-Message.Command.Help.Behavior=Toogles the behavior of the pet  (alias: /pb or /petb)
-Message.Command.Help.Call=Call your pet  (alias: /pc or /petc)
-Message.Command.Help.CaptureHelper=Toogles the Capturehelper on/off
-Message.Command.Help.ChooseSkilltree=Shows and chooses skilltrees  (alias: /pcst or /petcst)
-Message.Command.Help.Info=Display info about a MyPet  (alias: /pinfo)
+Message.Command.Help.Behavior=Selects the behavior of the pet  (alias: /pb or /petb)
+Message.Command.Help.Call=Calls your pet  (alias: /pc or /petc)
+Message.Command.Help.CaptureHelper=Toggles the CaptureHelper on or off
+Message.Command.Help.ChooseSkilltree=Shows and allows you to choose skilltrees  (alias: /pcst or /petcst)
+Message.Command.Help.Info=Displays info about a MyPet  (alias: /pinfo)
 Message.Command.Help.Inventory=Opens the inventory of the pet  (alias: /pi or /peti)
 Message.Command.Help.MoreInfo=For more info read the command page here:
-Message.Command.Help.Name=Set the name of your pet
-Message.Command.Help.Pickup=Toogles pickup on/off  (alias: /pp or /petp)
-Message.Command.Help.Release=Release your pet
-Message.Command.Help.Respawn=Let yout pet respawn for a charge (alias: /pr or /petr)
+Message.Command.Help.Name=Changes the name of your pet.
+Message.Command.Help.Pickup=Toggles item pickup on or off  (alias: /pp or /petp)
+Message.Command.Help.Release=Releases your pet
+Message.Command.Help.Respawn=Respawns your pet instantly, but with a fee (alias: /pr or /petr)
 Message.Command.Help.SendAway=Sends your pet away  (alias: /psa or /petsa)
 Message.Command.Help.Skill=Shows the skill-levels
-Message.Command.Help.Stop=MyPet stopps attacking  (alias: /ps or /pets)
-Message.Command.Help.Type=Shows info about MyPet-Types
-Message.Command.Name.New=The name of your pet is now: <aqua>{0}<r>
+Message.Command.Help.Stop=Stops your pet from attacking  (alias: /ps or /pets)
+Message.Command.Help.Type=Shows info about MyPet types
+Message.Command.Name.New=Your pet's name is now: <aqua>{0}<r>
 Message.Command.Name.ToLong="<r>{0}<r>" is too long. The maximum lenght is <gold>{1}<r> characters.
-Message.Command.PetType.Unknown=<red> Unknown Pettype!
-Message.Command.Release.Dead=<aqua>{0}<reset> is <red>dead<reset> and will never come back. . .
+Message.Command.PetType.Unknown=<red>Unknown pet type!
+Message.Command.Release.Dead=<aqua>{0}<reset> has been released, and is no longer alive...
```

Comment:
Even if the old one wasn't correct (IDK) it sounded way better. The released part may need some change.